packit/local_project.py

```diff
@@ -398,14 +399,18 @@ def checkout_ref(self, ref: str):
 
     def checkout_pr(self, pr_id: Union[str, int]):
         """
-        Fetch selected PR and check it out. This will work for github and pagure, not for gitlab.
+        Fetch selected PR and check it out.
         """
+        is_gitlab = True if isinstance(self.git_service, GitlabService) else False
+        forge = "merge-requests" if is_gitlab else "pr"
```

Comment:
What about having `pr` for all cases to have unified behaviour?

Reply:
At least, make it consistent -- for `pr` we should use `mr`.

Reply:
Yeah, this was not needed! pr will be fine.